models/app.model.js

```diff
@@ -1,9 +1,28 @@
 const db = require('../db/connection')
+const articles = require('../db/data/test-data/articles')
 
 exports.fetchTopics = () => {
     return db.query(`
     SELECT * FROM topics;
     `).then((topics) => {
         return topics.rows
     })
+}
+
+exports.fetchArticles = () => {
+    return db.query(`
+    SELECT title, topic, articles.author, articles.article_id, articles.created_at, articles.votes, COUNT(comments.article_id) AS comment_count
+    FROM articles
+    LEFT JOIN comments ON comments.article_id = articles.article_id
+    GROUP BY articles.article_id
+    ORDER BY articles.created_at DESC
+    ;
+    `).then((articles)=> {
+       const countToNums = articles.rows.map((article) => {
```

Comment:
It looks as though this logic is to coerce the string value for comment_count to a number, which should ideally be done as part of the SQL query for better efficiency. I would look at using CAST() or double colon notation (::) for changing it automatically to a number.